src/clinical/clinical-entities.ts

```diff
@@ -36,4 +34,18 @@ export interface Sample {
   submitterId: string;
 }
 
+export interface Treatment {
+  clinicalInfo?: ClinicalInfo;
+  chemotherapy: Array<Therapy>;
```

Comment:
the only difference between therapies is the name .. I'm thinking that we can generalize this .. a Treatment has a List of Therapies and each therapy has a name/type + clinical info

Reply:
hmm that's how I set it up at first, but then should `name/type` be constrain to an enum: ['hormoneTherapy', 'chemotherapy', ...] or is just `string` enough. I didn't want to define a SubClinicalEntity enum

Reply:
^ for that I mean in the mongoose document schema

Reply:
in mongoose  that can be a string

Reply:
cool, then I agree that makes more sense